apiary-metastore-events/kafka-metastore-events/pom.xml

```diff
@@ -24,10 +24,6 @@
 
 
   <dependencies>
-    <dependency>
```

Comment:
Why is this removed? Are you still getting it transitively view the `slf4j-log4j12` dependency below?

Reply:
indeed it is